File: app/src/main/java/com/neome/java/api/home/base/dto/DtoUserNotification.java
// neome.ai API - do not change
//

package com.neome.java.api.home.base.dto;

import com.neome.java.api.core.base.dto.NotificationCustomData;

import org.jetbrains.annotations.Nullable;

import java.util.Date;

@SuppressWarnings("unused")
public class DtoUserNotification
{
  public String body;

  @Nullable
  public Date createdOn;

  @Nullable
  public NotificationCustomData customData;

  public String id;

  @Nullable
  public Boolean isRead;

  public String title;
}

File: app/src/main/java/com/neome/api/home/base/dto/DtoUserNotification.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.home.base.dto

import com.neome.api.meta.base.Types.EnumDefnKindNotification
import com.neome.api.core.base.dto.NotificationCustomData

interface DtoUserNotification
{
  val body: String
  val createdOn: String?
  val customData: NotificationCustomData?
  val id: String
  val isRead: Boolean?
  val kind: EnumDefnKindNotification?
  val title: String
}